### PR TITLE
Add Google Analytics

### PR DESCRIPTION
**Pull request by caseypt**
* Add Google Analytics snippet to base template.
* Use context processor to add Google Analytics account
to request context based on environment settings.

**Testing instructions:**
- Verify that the Google Analytics snippet is present on the homepage and the project page when visited directly.
- Verify that the account number is populated in the Google Analytics snippet and matches the value from the `development.py` settings file.

Connects to #698

### CODE REVIEW

File: src/mmw/apps/core/templates/base.html [hunk 1]
```diff
@@ -12,6 +12,16 @@
     <link rel="icon" type="image/png" href="{% static 'favicon@2x.png' %}" sizes="32x32">
     <link rel="stylesheet" href="{% static 'css/vendor.css' %}" />
     <link rel="stylesheet" href="{% static 'css/main.css' %}" />
+    <!-- Google Analytics -->
+    <script>
+        (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
```
Comment (lewfish) [inline src/mmw/apps/core/templates/base.html]: Strange how the minified arguments spell out 'isogram'.

File: src/mmw/apps/home/views.py [hunk 2]
```diff
@@ -123,7 +124,7 @@ def get_client_settings(request):
 
 
 def get_context(request):
-    context = {}
+    context = RequestContext(request)
```
Comment (lewfish) [inline src/mmw/apps/home/views.py]: Now I know how context processors work.

**Comment by lewfish** [comment]
+1, tested.

File: src/mmw/apps/core/templates/base.html (hunk 1 above)
Comment (mmcfarland) [inline src/mmw/apps/core/templates/base.html]: https://github.com/shinnn/isogram

**Pull request merged**